src/main.c

```diff
@@ -19,6 +19,7 @@ GMainLoop *r_loop = NULL;
 int r_exit_status = 0;
 
 gboolean info_noverify = FALSE;
+gchar *status_format = NULL;
```

Comment:
This should be called output_format, because it will be used for the "status" and "info" commands.